README.md

```diff
@@ -27,17 +27,17 @@ Download [the JARs][]. Or for Maven with non-Android, add to your `pom.xml`:
 <dependency>
   <groupId>io.grpc</groupId>
   <artifactId>grpc-netty</artifactId>
-  <version>1.0.0</version>
+  <version>1.0.1</version>
```

Comment:
These just got updated via a cherry-pick from the v1.0.x branch via #2315. Yeah, I know that 1.0.1 had been out for a while, but we hadn't done the notifications and such until today.

Thank you for sending out these changes, because they did provide a nudge to finish the release.